docs/website/docs/command-reference/create-namespace.md

```diff
@@ -10,10 +10,7 @@ To create a namespace you can run `odo create namespace <name>`:
 ```shell
 odo create namespace mynamespace
 ```
-
-Example - 
 ```shell
-odo create namespace mynamespace
```

Comment:
I am not sure about this. Currently, both the section about command and output seem unrelated to each other.
I would rather keep this as the following -
```sh
$ odo create namespace mynamespace
 ✓  Namespace "mynamespace" is ready for use
 ✓  New namespace created and now using namespace: mynamespace
```

Same comment for all the changes.